### PR TITLE
can add comment, with message saying when it is posting and greyed ou…

### PR DESCRIPTION
**Pull request by NRMSMITH**
…t box, message for an error and messaged for successful post

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -16,6 +17,7 @@ function App() {
         <Route path="/" element={<Articles />} />
         <Route path={`/topics/:topic_name`} element={<Topics />} />
         <Route path={`/articles/:article_id`} element={<SingleArticle />} />
+        <Route path={`/articles/:article_id/addcomment`} element={<AddNewComment />} />
```
Comment (xmenbteam) [inline src/App.js]: Do you need this path now that your add comment component is on the same page?

File: src/Pages/SingleArticle.jsx [hunk 2]
```diff
@@ -57,7 +59,10 @@ return (
               />
             ); 
         })}
-    </section>
+        
+            <AddNewComment article={singleArticle} topics={topics} setComments={setComments}/>
```
Comment (xmenbteam) [inline src/Pages/SingleArticle.jsx]: Should this be above comments? It's much nicer to see the box at the top :)

**Review by xmenbteam — APPROVED** [review]
This is great! Your comments are coming in in the wrong order though... They should come back most recent first. This is an easy fix in your back end or you can do it through a query in your front end. Either way it's an easy fix!

File: src/App.js (hunk 1 above)
Comment (NRMSMITH) [reply src/App.js]: done! Completely forgot about it!

File: src/Pages/SingleArticle.jsx (hunk 2 above)
Comment (NRMSMITH) [reply src/Pages/SingleArticle.jsx]: Done! Just working on the order of the comments!

**Pull request merged**